docs/deprecated.md

```diff
@@ -266,7 +266,7 @@ of the `--changes` flag that allows to pass `Dockerfile` commands.
 
 ### Interacting with V1 registries
 
-**Disabled By Default In Release: v17.04**
+**Disabled By Default In Release: v17.05**
```

Comment:
Let me move it to 17.06; we can decide on the change in policy separate from that